config/registry/google/modules/gcp-k8s-service.yaml

```diff
@@ -31,12 +31,14 @@ inputs:
     description: "Manually set secrets"
     default: [ ]
   - name: read_buckets
-    user_facing: false
-    description: Buckets linked for reading
+    user_facing: true
+    description: Buckets to grant read permissions for
```

Comment:
I don’t think this should be exposed to the user. If the user wants to add permissions for external buckets they should use “extra_project_iam_roles”